### ISSUE TITLE
Detect language changes  changes during read aloud

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/epub-accessibility-tests#82]
We have detect language changes when using screen readers, and we want to have the same test, but for Read Aloud.  We thought this should be experimental.

**Comment by GeorgeKerscher** [comment]
On our Reading System call today, September 30, 2025, this issue came up. Reading Systems should be able to use other languages. Being able only to read English content is a failure. 

We thought this should be moved into the fundamental Read Aloud test book.

I suspect that we will mirror the language change test in the Non-visual reading test book.

**Comment by ways2read** [comment]
This was a candidate for the last revision. It did not make the cut because few readaloud implementations supported language switching.

I believe that read aloud is commonly supported for non-English languages by using the voice selected by the user. A user in Germany probably has system TTS voice set to German. But this test would be about supporting language changes indicated in the content.

Richard
